src/coreclr/jit/assertionprop.cpp

```diff
@@ -3221,6 +3221,7 @@ GenTree* Compiler::optVNConstantPropOnTree(BasicBlock* block, GenTree* tree)
         }
 
         case TYP_SIMD32:
+        case TYP_SIMD64: // TODO-XArch-AVX512: Fix once GenTreeVecCon supports gtSimd64Val.
```

Comment:
We should also make sure there is an explicit issue in GitHub tracking this todo before merging

Reply:
Created issue - https://github.com/dotnet/runtime/issues/82312

I do not have permission to
1. Create correct tags for this.
2. Add this to https://github.com/dotnet/runtime/issues/77034

Will need help

Reply:
Done. Thanks!